android/build.gradle

```diff
@@ -50,6 +50,6 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.1'
     implementation 'net.sourceforge.streamsupport:streamsupport-cfuture:1.7.4'
-    implementation 'org.torusresearch:web3auth-android-sdk:3.4.0'
+    implementation 'com.github.web3auth:web3auth-android-sdk:3.6.0'
```

Comment:
why is this changed to com.github?

Reply:
With org.torusresearch, its not working. When changed to com.github.web3auth, it starts working.